tests/components/fritz/test_config_flow.py

```diff
@@ -761,3 +765,29 @@ async def test_ssdp_ipv6_link_local(hass: HomeAssistant) -> None:
     )
     assert result["type"] is FlowResultType.ABORT
     assert result["reason"] == "ignore_ip6_link_local"
+
+
+async def test_upnp_not_enabled(hass: HomeAssistant) -> None:
+    """Test if UPNP service is enabled on the router."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_USER, "show_advanced_options": False},
```

Comment:
Isn't show advanced options -> false the default?

Reply:
Honestly I didn't check ;-)

Reply:
Removed